node/tests/ServerModules.test.ts

```diff
@@ -1694,5 +1694,195 @@ describe("Server Module Tests", () => {
             ];
             expect(stringResult).toEqual(expectedStringResult);
         });
+
+        it("FT.ALIASADD test", async () => {
```

Comment:
You can test all 3 in a single test without repeating `create` and `aliasadd` every time

Reply:
Updated.